gensim/test/test_doc2vec.py

```diff
@@ -168,15 +171,15 @@ def model_sanity(self, model):
     def test_training(self):
         """Test doc2vec training."""
         corpus = DocsLeeCorpus()
-        model = doc2vec.Doc2Vec(size=100, min_count=2, iter=20)
+        model = doc2vec.Doc2Vec(size=100, min_count=2, iter=20, window=8, sample=0.01, workers=1)
```

Comment:
Working with a tiny corpus, `workers=1` is a good idea (and I believe what most of the Word2Vec tests do. Were the other two parameter changes also necessary for the test to pass? (If not, better to test the new defaults.)